### PR TITLE
Syscoin to azure devtestlab

### PR DESCRIPTION
**Pull request by sidhujag**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@sidhujag__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

File: Artifacts/linux-syscoin/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+* Click the `Deploy to Azure` icon above
+* Complete the template parameters, choose your resource group, accept the terms and click Create
+* Wait about 15 minutes for the VM to spin up and install the software
+* Connect to the VM via SSH using the DNS name assigned to your Public IP
+* If you wish to relaunch syscoind `sudo syscoind`
+* syscoind will run automatically on restart
+
+# Licensing
+
+Syscoin is released under the terms of the MIT license. See `COPYING` for moreinformation or see http://opensource.org/licenses/MIT.
```
Comment (drwill-ms) [inline Artifacts/linux-syscoin/README.md]: Minor: looks like you need a space between "more" and "information".

File: Artifacts/linux-syscoin/build_syscoin.sh [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+if [ $1 = 'From_Source' ]; then
+#################################################################
+# Build Syscoin from source                                     #
+#################################################################
+NPROC=$(nproc)
+echo "nproc: $NPROC"
+#################################################################
+# Install all necessary packages for building Syscoin           #
+#################################################################
+sudo apt-get -y install git build-essential libtool autotools-dev autoconf pkg-config libssl-dev libevent-dev bsdmainutils libboost-all-dev libminiupnpc-dev libzmq3-dev
```
Comment (drwill-ms) [inline Artifacts/linux-syscoin/build_syscoin.sh]: Not all Linux distros have apt-get. In other artifacts we've added conditional support for apt-get or yum. Here's an example: https://github.com/Azure/azure-devtestlab/blob/master/Artifacts/linux-install-mongodb/linux_install_mongodb_v1.sh

**Comment by drwill-ms** [comment]
Sorry about the delay on your PR sidhujag. I'll get some of our more Linux-savvy team members to take a look.

Can you speak to the testing you've done with this artifact? Which Linux gallery images have you applied it to?

File: Artifacts/linux-syscoin/Artifactfile.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "https://raw.githubusercontent.com/Azure/azure-devtestlab/master/schemas/2015-01-01/dtlArtifacts.json",
+  "title": "Syscoin (Business on the blockchain)",
```
Comment (drwill-ms) [inline Artifacts/linux-syscoin/Artifactfile.json]: We like to have simplified titles. Will just "Syscoin" do? More clarification can go into the description.

**Comment by azurecla** [comment]
__@sidhujag__, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, AZPRBOT;

**Comment by sidhujag** [comment]
@drwill-ms those repositories differ on the different OSes right? for example how do we install the ppa for bitcoin in fedora? I've tested with ubuntu 15.10 / 14.04 LTS and ran the sh script to ensure it makes and runs the artifact output file.

File: Artifacts/linux-syscoin/build_syscoin.sh (hunk 2 above)
Comment (ryanvog) [inline Artifacts/linux-syscoin/build_syscoin.sh]: To support inclusion in the public artifact repository, you need to support Debian (apt-get) and Fedora (yum) package managers.  Also, you will need to ensure your artifact supports the following Azure-based linux distros: Ubuntu, Fedora RedHat, CoreOS, CentOS, and SuSE.  Look at some of the other linux artifacts in the repository to see examples on how to do this.

Alternatively, host your own public-based artifact repository.

**Comment by mmydland** [comment]
This hasn't been updated at all since March so I am closing it down

**Comment by sidhujag** [comment]
We definitely want devtestlab integration for Syscoin as we are currently in the process of certifying Syscoin blockchain services from within BaaS Azure marketplace.. Can you tell me what I need to do to have it merged?

**Comment by mmydland** [comment]
I think we just wanted the changes that Ryan suggested for supporting yum along with apt-get.  If we can get that change in, we will be happy to merge it.  Reopening the PR.

**Comment by drwill-ms** [comment]
No action on PR feedback, so I'm closing this PR again. Feel free to resubmit at a future date if you can perform the actions requested.

**Pull request closed without merging**